test/e2e/framework/nodes_util.go

```diff
@@ -64,6 +73,15 @@ func etcdUpgradeGCE(target_storage, target_version string) error {
 	return err
 }
 
+func ingressUpgradeGCE() error {
+	// Flip glbc image from latest release image to HEAD to simulate an upgrade.
+	// Kubelet should restart glbc automatically.
+	sshResult, err := NodeExec(GetMasterHost(), "sudo sed -i -re 's/(image:)(.*)/\\1 gcr.io\\/e2e-ingress-gce\\/ingress-gce-e2e-glbc-amd64:latest/' /etc/kubernetes/manifests/glbc.manifest")
```

Comment:
Should the name of the container be more unique? Isn't this going to end up sharing the same image across all invocations of the test (say on my personal machine...)

Reply:
Ack. Issue to track that change is #6381